utils/framework/filter.py

```diff
@@ -49,6 +49,13 @@ async def find_triggered_filters(input, member: discord.Member) -> List[FilterWo
             words_found.append(word)
     return words_found
 
+def ignorable_words(triggered_filter_words: List[FilterWord]):
```

Comment:
`ignorable_words` doesn't sound like a very descriptive name, maybe `has_only_silent_words`

Or if you want, you can flip the logic (return true if at least one word is not silently filtered) and call it something different like `has_notifiable_word`

Reply:
Thanks, I've renamed it to `has_only_silent_filtered_words`